packages/storybook/README.md

```
@@ -0,0 +1,83 @@
+</p>
+
+
+`@faststore/storybook` provides an isolated environment for developing, testing, and documenting FastStore UI components. It is an internal development tool — not published to npm.
+
+Stories import components from `@faststore/components` and are styled via `@faststore/ui`, making it a live preview of both the structure and the visual output of the component library.
+
+> **Note:** Stories coverage is currently incomplete. Only a subset of components in `@faststore/components` have stories. Adding stories for missing components is welcome.
+
+## Package structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)